README.md

```diff
@@ -1 +1,359 @@
+docker cp "$DATA_PATH/datasets.json" deploy-db-1:tmp/datasets.json
+docker cp "$DATA_PATH/genomicVariationsVcf.json" deploy-db-1:tmp/genomicVariations.json
+docker cp "$DATA_PATH/individuals.json" deploy-db-1:tmp/individuals.json
+docker cp "$DATA_PATH/runs.json" deploy-db-1:tmp/runs.json
+```
+Then:
+```bash
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/datasets.json --collection datasets
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/analyses.json --collection analyses
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/biosamples.json --collection biosamples
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/cohorts.json --collection cohorts
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/genomicVariations.json --collection genomicVariations
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/individuals.json --collection individuals
+docker exec deploy-db-1 mongoimport --jsonArray --uri "mongodb://root:example@127.0.0.1:27017/beacon?authSource=admin" --file /tmp/runs.json --collection runs
+```
+
+And finally
+```bash
+docker exec beacon python beacon/reindex.py
+```
+To create indexes for the database.
+
+You then need to set up ontologies correctly with:
+```bash
+docker exec beacon python beacon/db/extract_filtering_terms.py
+docker exec beacon python beacon/db/get_descendants.py
+```
+
+## Create endpoint for public crypt4gh key
+Create a new bucket and upload the key to that bucket. Then change the key permissions to public:
+```bash
+s3cmd -c <s3config> mb s3://gdi-public
+s3cmd -c <s3config> put c4gh.pub.pem s3://gdi-public/key/
+s3cmd -c <s3config> setacl s3://gdi-public/key/c4gh.pub.pem --acl-public
+```
+
+If you are using the Swedish GDI implementation, the public key for the next step can be found under `https://s3.sto3.safedc.net/gdi-public/key/c4gh.pub.pem`
+
+## Uploading data to the archive
+
+From the VM where the storage-and-interfaces is deployed, extract the crypt4gh public key from the download container using:
+```bash
+$ docker cp download:/shared/c4gh.pub.pem .
+```
+
+Download the `c4gh.pub.pem` to the machine where the dataset exists.
+
+Login to the auth endpoint (https://login.gdi.nbis.se) and download the s3config file containing the JWT token.
+Download and extract the sda-cli
+```bash
+$ wget https://github.com/NBISweden/sda-cli/releases/download/v0.0.6/sda-cli_.0.0.6_Darwin_x86_64.tar.gz
+$ tar -xvzf sda-cli_.0.0.6_Darwin_x86_64.tar.gz
+```
+
+Encrypt and upload the file using the sda-cli
```

Comment:
```suggestion
Encrypt and upload the file using `sda-cli`
```

This could obviously also be done using `sda-admin`,  and that might even be simpler?